star_tides/core/actions/create_user_action.py

```diff
@@ -30,15 +30,11 @@ def __init__(self, first_name, last_name, email, password):
     def run(self):
         salt = bcrypt.gensalt()
         self.password = self.password.encode('utf-8')
-
         pw_hash = bcrypt.hashpw(self.password, salt)
-
-        user = User(
-            first_name=self.first_name,
-            last_name=self.last_name,
-            email=self.email,
-            password=pw_hash
+        user = UserInterface.create_user(
```

Comment:
s/UserInterface/UserStorage?

Reply:
Does create_user write to the DB? Might be worth renaming to build_user if not.

Reply:
My intention with the naming is that the `UserInterface` is an interface to the `User` collection in the DB. As far as the `create_user` vs `build_user`, I think they both convey the point. I just went with `create` based on the `CRUD` acronym. That support fails when it comes to reading a user though because I would call that `get_user`.